.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,49 @@
+name: Update Ansible SDK docs
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  sync:
```

Comment:
If you're going to be using secrets, make sure to and an `environment` entry here and keep those secrets under the environment in the repo settings rather than having them available globally. It also allows you to specify the `url` key under the environment section.